redis/reply.go

```diff
@@ -477,3 +477,44 @@ func Positions(result interface{}, err error) ([]*[2]float64, error) {
 	}
 	return positions, nil
 }
+
+func Uint64s(reply interface{}, err error) ([]uint64, error) {
```

Comment:
Small nit, all the other `XXXs(...)` methods are together, same with the `XXXMap(...)` so be nice to keep each of these in the same area too.